inc/ticket.class.php

```diff
@@ -830,4 +853,83 @@ static function filter_actors(array $params = []): array {
 
        return $params;
     }
+
+    public static function addToTimeline($options)
+    {
+        $item = $options['item'];
+        $group = new Group_Ticket();
+        $groupUsed = $group->find([
+            'tickets_id' => $options['item']->getID(),
+            'type' => CommonITILActor::ASSIGN,
+        ]);
+        if (empty($groupUsed)) {
+            $groupUsed = 0;
+        } else {
+            $temp = $groupUsed;
+            $groupUsed = [];
+            foreach ($temp as $grp) {
+                $groupUsed[] = $grp['groups_id'];
+            }
+        }
+        $entity = $options['item']->fields['entities_id'];
+        $listEntities = getSonsOf(Entity::getTable(), $entity);
+        if (empty($listEntities)) {
+            $listEntities = $entity;
+        }
+        $criteria = [
+            'OR' => [
+                'AND' => [
+
+                    'entities_id' => $listEntities,
+                    'is_recursive' => 1
+                ],
+                ['entities_id' => $entity],
+            ],
+            'is_assign' => 1,
+            'AND' => [
+                'NOT' => [
+                    'id' => $groupUsed
+                ]
+            ]
+        ];
+        $groupBis = new Group();
+        $groups = $groupBis->find($criteria);
+
+        $itemtypes = [];
+
+        $ticket_task = new PluginEscaladeTicket();
+        if (!empty($groups)) {
+            $itemtypes['escalation'] = [
+                'type' => 'PluginEscaladeTicket',
+                'class' => 'action-escalade',
+                'icon' => 'fas fa-comments',
```

Comment:
```suggestion
                'icon' => ti ti-arrow-up',
```
Would be more consistent with the icon used here:

![image](https://user-images.githubusercontent.com/42734840/228575124-e06d9b58-f593-44cb-b695-934de14a0243.png)